src/Type/Integer.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Integer value object class.
+ *
+ * Implemented as an immutable object with a pair of named constructors.
+ */
+
+namespace Joomla\Webservices\Type;
+
+final class Integer extends Type
```

Comment:
Int and Integer classes that looks exactly the same, maybe one should be kicked out